src/main/resources/forge.sas

```diff
@@ -106,4 +106,5 @@ net/minecraft/world/biome/BiomeAmbience func_235220_d_()Ljava/util/Optional; # g
 net/minecraft/world/biome/BiomeAmbience func_235222_e_()Ljava/util/Optional; # getAmbientSound
 net/minecraft/world/biome/BiomeAmbience func_235224_f_()Ljava/util/Optional; # getMoodSound
 net/minecraft/world/biome/BiomeAmbience func_235226_g_()Ljava/util/Optional; # getAdditionsSound
-net/minecraft/world/biome/BiomeAmbience func_235228_h_()Ljava/util/Optional; # getMusic
\ No newline at end of file
+net/minecraft/world/biome/BiomeAmbience func_235228_h_()Ljava/util/Optional; # getMusic
+net/minecraft/network/play/server/SEntityPropertiesPacket func_149441_d()Ljava/util/List; # getSnapshots
```

Comment:
I believe the checkSAS task was forgotten here

Reply:
Nope. I ran it. Ran it again just now, it did not change anything and did not report any errors.

Reply:
Odd I thought the SAS file was supposed to be alphabetized just like the AT file.

Reply:
So did I.

Reply:
It isn't done automatically. Its one of the todo's. But if you would be so kind, putting them in the correct place just makes it easer to read.

Reply:
It's ordered now, I didn't mean to come across as sassy